lblod/frontend-embeddable-notule-editor
# GN-4820: Bump editor and enable copy/pasting variable nodes correctly…

### Overview

GN-4820: Bump editor and enable copy/pasting variable nodes correctly

##### connected issues and PRs:

* https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/435
* https://github.com/lblod/ember-rdfa-editor/pull/1195


### Setup

1. Checkout
2. `pnpm i` 

### How to test/reproduce

See if copy/pasting variable works


https://github.com/lblod/frontend-embeddable-notule-editor/assets/769698/6e8de392-8fa5-477f-86df-854fcaba4aec





### Checks PR readiness
